{"name": "困人处置列表", "status": "passed", "attachments": [{"name": "log", "source": "f3a5b6b3-1c13-4b53-b027-e27eb239accd-attachment.txt", "type": "text/plain"}], "start": 1690256287522, "stop": 1690256287667, "uuid": "2aa6da48-a20a-471f-ab3f-a4c464158ce7", "historyId": "e9533ca1bce9dcd222ffc25dce3a836e", "testCaseId": "1307c5a2fbf84e6aa7a4e7b10201eae2", "fullName": "opsPC.test_emergency#test_device_error_trapped_list_001", "labels": [{"name": "feature", "value": "运维管理平台"}, {"name": "story", "value": "应急处置"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "18760-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}